java/light-4j/pom.xml

```diff
@@ -0,0 +1,428 @@
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.light-4j</groupId>
+    <artifactId>benchmark</artifactId>
+    <packaging>jar</packaging>
+    <name>benchmark</name>
+    <version>0.0.1</version>
+
+    <properties>
+        <java.version>1.8</java.version>
+        
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.main.class>com.networknt.server.Server</project.main.class>
+        <version.light-4j>1.6.23</version.light-4j>
```

Comment:
could you use a range like `[1.6,1.7)` to enforce using last release in **1.6.x**

Reply:
Done in b81fb2691f80630445b1add4f978e939c0180966.